src/submission/submission-api.ts

```diff
@@ -386,4 +433,18 @@ const isValidCreateBody = (req: Request, res: Response): boolean => {
   return true;
 };
 
+const isValidRequestArgs = (programId: string, uploadedFiles: {}): boolean => {
+  if (programId === undefined) {
+    L.debug('programId missing');
+    // ControllerUtils.badRequest(res, `programId is required`);
+    return false;
+  }
+  if (/*uploadedFiles === undefined &&*/ Object.keys(uploadedFiles).length === 0) {
```

Comment:
Looks like some test code that wasn't removed?

Reply:
Cleared it. Thanks.